corehq/apps/app_manager/suite_xml/case_tile_templates/clinic_and_unit.json

```diff
@@ -0,0 +1,5 @@
+{
+    "slug": "clinic_and_unit",
```

Comment:
suggestion: `parent_and_child`? Same rationale as giving the field generic names